clang/test/OpenMP/openmp_attribute_parsing.cpp

```diff
@@ -49,8 +49,8 @@
 
 // Test that we give a sensible error on an unknown attribute in the omp
 // namespace that has an argument list.
-[[omp::unknown()]]; // expected-warning {{unknown attribute 'unknown' ignored}}
-[[using omp: unknown()]]; // expected-warning {{unknown attribute 'unknown' ignored}}
+[[omp::unknown()]]; // expected-warning {{unknown attribute 'omp::unknown' ignored}}
+[[using omp: unknown()]]; // expected-warning {{unknown attribute 'omp::unknown' ignored}}
```

Comment:
When we get around to doing fixes and typo corrections, this will be an interesting test case to keep in mind, especially for an unknown vendor namespace.